packages/unlock-js/src/web3Service.ts

```diff
@@ -905,18 +905,35 @@ export default class Web3Service extends UnlockService {
   async referrerFees({
     lockAddress,
     network,
-    address,
   }: {
     lockAddress: string
     network: number
-    address: string
   }) {
-    const lockContract = await this.getLockContract(
-      lockAddress,
-      this.providerForNetwork(network)
+    const query = `{
+      lock(id:"${lockAddress}") {
+        referrerFees {
+          id
+          referrer
+          fee
+        }
+      }
+    }`
+
+    const result = await fetch(
```

Comment:
The `referrerFees` function retrieves all `referrerFees` from a "lock" address.

This code will be updated once the graph is uploaded to the original "Unlock Protocol" subgraph on all networks. Afterwards, I will pull in the graph URLs based on the network using import networks from `@unlock-protocol/networks` ✅.